styles.css

```diff
@@ -4,10 +4,11 @@
 body{
   margin:0;
   display: flex;
-  flex-wrap: wrap;
+  flex-wrap:wrap;
   font-family
 }
 header, footer, aside, .main{
+  margin: 0;
```

Comment:
👍 for specifically direct children, would want a rule with selector `header>*, footer>*, aside>*, .main>*`